src/Processors/Formats/Impl/PrettyBlockOutputFormat.cpp

```diff
@@ -482,6 +501,12 @@ void PrettyBlockOutputFormat::writeMonoChunkIfNeeded()
 {
     if (mono_chunk)
     {
+        if (mono_chunk.getNumRows() > format_settings.pretty.max_rows)
+        {
+            if (mono_chunk.getNumRows() > 10)
```

Comment:
Please name the constant, e.g. `constexpr size_t last_rows_num = 10`
Alternatively, `auto last_rows_num = std::min(format_settings.pretty.max_rows / 2, 10)`